src/InputOutput/VTK/VTK.jl

```diff
@@ -6,4 +6,8 @@ include("writemesh.jl")
 include("writevtk.jl")
 include("writepvtu.jl")
 
+function __init__()
```

Comment:
Won't this lead to the multiple init issue I mentioned on the call?
https://github.com/climate-machine/CLIMA/pull/527/files#diff-0c102a9f3b330c12cac54a2a4821a48dR576

Should `tic`/`toc`s be scoped by the module they are defined in?

Reply:
The `tictoc()` implementation won't re-create timing infos that have already been created -- that's how we the multiple init problem is handled. Or am I forgetting something?

Reply:
Yea, you are right I am still a bit uncomfortable about namespace clashes, but most likely not something to worry about,